ddtrace/monkey.py

```diff
@@ -200,7 +201,15 @@ def patch(raise_errors=True, **patch_modules):
     )
 
 
+@deprecated(
+    message="This function will be removed.",
+    version="1.0.0",
+)
 def patch_module(module, raise_errors=True):
+    return patch_module(module, raise_errors=raise_errors)
+
+
+def _patch_module(module, raise_errors=True):
```

Comment:
we need to update this function to call `_attempt_patch_module` instead of `_patch_module`, otherwise it will infinitely call itself.